collection_requirements.rst

```diff
@@ -98,24 +98,24 @@ Modules & Plugins
 
 Documentation
 ~~~~~~~~~~~~~~
-All module and plugin ``DOCUMENTATION`` and ``RETURN`` MUST:
 
-* Use the FQCN for ``M(...)`` and ``- module:`` references of ``seealso`` subsections. See `Linking within module documentation <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation>`_
-* Use the field ``version_added`` to document the version of the collection for which an option, module or plugin was added.
+All modules and plugins MUST:
 
-  * Use collection version numbers for ``version_added``, and not Ansible version numbers or other unrelated version numbers.
-  * If you for some reason really have to specify version numbers of Ansible or of another collection, you have to provide ``version_added_collection``. We strongly recommend to NOT do this.
-  * Not every option, module or plugin must have ``version_added``. You should use it to mark when new content (modules, plugins, options) were added to the collection. The values are shown in the documentation, and this can be very useful for your users.
+* Have a `DOCUMENTATION <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-block>`_ block
+* Have an `EXAMPLES <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#examples-block>`_ block (except where not relevant for the plugin type)
+* Use FQCNs (including ansible.builtin) when referring to modules, plugins and documentation fragments inside and outside the collection
 
-All module and plugin ``EXAMPLES`` MUST:
+When using ``version_added`` in the documentation:
 
-* Use FQCN for module (or plugin) name.
-* For modules (or plugins) left in ansible-base use ``ansible.builtin.`` as a FQCN prefix, for example, ``ansible.builtin.template``
+* It must be used to declare the version of the collection in which the options were added -- NOT the version of Ansible.
+* If you for some reason really have to specify version numbers of Ansible or of another collection, you have to provide ``version_added_collection``. We strongly recommend to NOT do this.
```

Comment:
```suggestion
* If you for some reason really have to specify version numbers of Ansible or of another collection, you also have to provide ``version_added_collection: collection_name``. We strongly recommend to NOT do this.
```
@felixfontein could you confirm? I haven't found any examples in my local forks

Reply:
Yes, it's correct. Ansible automatically adds `version_added_collection` for the version the fragment comes from, but if you add it manually it won't overwrite it.

Reply:
@Andersson007 it's used is a couple places: https://codesearch.recordsansible.org/?q=version_added_collection

Reply:
@dmsimard ok, thanks!